api/valor_api/schemas/types.py

```diff
@@ -29,110 +22,90 @@
 ]
 
 
-def _check_if_empty_annotation(annotation: "Annotation") -> bool:
+def _match_annotation_to_implied_task_type(
```

Comment:
I feel that storing this implied data type would be very useful for filtering. It doesnt have to be exposed by the API (besides maybe DatasetSummary)

Reply:
I could get behind this idea, but let's save it for a separate PR that focuses on filtering enhancements

Reply:
I think too much functionality has been removed to approve, there needs to be a way to determine the data type in filters.